src/aosm/README.md

```diff
@@ -85,8 +85,14 @@ For CNFs, you must provide helm packages with an associated schema. When filling
             ]
         },
 
+#### NSDs
+
+For NSDs, you will need to have a Resource Group with a deployed Publisher, Artifact Store, Network Function Definition and Network Function Definition Version. You can use the `az aosm nfd` commands to create the Network Function Definition Version resource.
```

Comment:
You can also use the az aosm nfd publish (might be wrong command) to make the publisher,resource group and artifact manifests (for vnfs). Maybe add that if you are intending to use VNFs, you can get all the prereqs done with the publish command.